future/future_test.go

```diff
@@ -0,0 +1,33 @@
+package future
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var expectations = []expectation{
+	{
+		// we should NOT see the file third_party/go-algorand/ledger/ledgercore/accountdata.go:
+		"../third_party/go-algorand/ledger/ledgercore/accountdata.go",
+		missing{shouldMiss: true, msg: "\n!!!path ledger/ledgercore/accountdata.go detected ==> unlimited assets has arrived\n"},
+
+		// AND struct AccountBaseData should not be available either:
+		"AccountBaseData",
+		missing{shouldMiss: true, msg: "\n!!!struct AccountBaseData is detected ==> unlimited assets has arrived\n"},
+
+		// AND finally, AccountBaseData.TotalAssets also should not exist:
+		[]string{"TotalAssets"},
+		missing{shouldMiss: true, msg: "\n!!!field TotalAssets is detected ==> unlimited assets has arrived\n"},
+	},
+}
```

Comment:
These are the assertions any of which becoming true will fail the tests.

Reply:
Nice trick! Is it still needed if we don't add MinBalance?

Reply:
Do we still need it? Only if the merging of `unlimited-assets` requires immediate attention elsewhere in indexer. According the [REST API doc](https://docs.google.com/document/d/11Hm21Dd1MKi7_DVNdrCo1IdfkffCx1_WojYyjByubjA/edit#heading=h.2gazcsgmxkub) I could see others as well.

But I'll be first to admit that this is all a bit silly. With proper communication, we don't need such a test. Once we know what to test for, we know what we should be looking out for. So what's the point of such a test?

Reply:
I think you're right, lets remove this.